README.md

```diff
@@ -15,9 +15,9 @@ Mermaid is a Javascript based diagramming and charting tool that uses Markdown-i
 
 Diagramming and documentation costs precious developer time and gets outdated quickly.
 But not having diagrams or docs ruins productivity and hurts organizational learning. <br/>
-Mermaid addresses this problem by cutting the time, effort and tooling that is required to create modifiable diagrams and charts, for smarter and more reusable content.
+Mermaid addresses this problem by cutting the time, effort and tooling that is required to create a modifiable diagrams and charts, for smarter and more reusable content.
```

Comment:
Since _diagrams_ and _charts_ are plural, the current form should be correct and no _a_ required.